evap/student/templates/student_vote.html

```diff
@@ -188,10 +188,22 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             function isTextareaInvalid(textarea) {
                 return textarea.val().length > 0 && ["", "ka", "na"].includes(textarea.val().replace(/\W/g,'').toLowerCase());
             }
+            // if it is not invalid, maybe show a reference warning (because referencing other answers is bad)
+            function doesTextareaContainReference(textarea) {
+                return ["s.o.", "siehe oben", "wie oben", "see above", "bereits erwähnt", "already stated"]
+                    .some(ref => textarea.val().toLowerCase().includes(ref));
```

Comment:
if i understand correctly the `textarea.val().toLowerCase()` part is done per element in the array, i would suggest to do that only once.